docs/application/dotnet/guides/machine-learning/singleshot.md

```diff
@@ -91,33 +96,32 @@ TensorsData in_data = in_info.GetTensorsData();
 in_data.SetTensorData(0, in_buffer);
 ```
 
-After preparing the input data, you can invoke the model and get the inference output result. The `SingleShot.Invoke()` method gets the input data to be inferred as a parameter and returns the `Tizen.MachineLearning.InferenceTensorsData` instance, which contains the inference result:
+After preparing the input data, you can invoke the model and get the inference output result. The `SingleShot.Invoke()` method gets the input data to be inferred as a parameter and returns the `Tizen.MachineLearning.Inference.TensorsData` instance, which contains the inference result:
 
 ```C#
 /* Invoke the model and get the inference result */
 TensorsData out_data = single.Invoke(in_data);
 ```
 
 <a name="fetch"></a>
-## Fetching Inference Result
+## Fetching the inference result
```

Comment:
Global Comment: 
Thank you for updating the headings in sentence case.
Please do not add extra articles in the heading.
It is recommended to use "sentence case" and "verb format" for headings. Please do not use the gerund form (ing form).
For example, 
1. Please change:
## Fetching Inference Result
to
## Fetch inference result